module.jai

```diff
@@ -495,7 +780,7 @@ write_string_value :: (col: int, name: string, info: *Type_Info, slot: *void, co
 }
 
 write_integer :: (col: int, name: string, info: *Type_Info, pointer: *void, col_type: Pq_Type, value: s64) -> bool {
-    if info.type != Type_Info_Tag.INTEGER {
+    if info.type != Type_Info_Tag.INTEGER && info.type != Type_Info_Tag.ENUM {
```

Comment:
This is not correct! `info` gets cast to `Type_Info_Integer` below, which is not OK if it is actually a `Type_Info_Enum`. This would re-interpret the name pointer as the `signed` property!